server/joligen_api.py

```diff
@@ -1,13 +1,20 @@
-from fastapi import Request, FastAPI, HTTPException
+from fastapi import Request, FastAPI, HTTPException, WebSocket
 import asyncio
 import traceback
 import json
 import subprocess
 import os
 import shutil
+from pathlib import Path
+import time
 
 import torch.multiprocessing as mp
 
+try:
+    mp.set_start_method("spawn")
```

Comment:
What is this for? I don't remember

Reply:
I kept this related error log from november 2023, it starts `torch.multiprocessing` with `spawn` method if it's not been already started: 

> RuntimeError: Cannot re-initialize CUDA in forked subprocess. To use CUDA with multiprocessing, you must use the 'spawn' start method